docs/source/dataframe.rst

```diff
@@ -42,13 +42,13 @@ The ``divisions`` attribute, analogous to ``chunks`` in ``dask.array`` is
 particularly important.  The values in divisions determine a partitioning of
 left-inclusive / right-exclusive ranges on the index::
 
-    divisions -- (10, 20, 40)
-    ranges    -- (-oo, 10), [10, 20), [20, 40), [40, oo)
+    divisions -- (0, 10, 20, 40, 50)
+    ranges    -- [0, 10), [10, 20), [20, 40), [40, 50)
```

Comment:
Does the last interval need to be closed on the right? Presumably `max(divisions) in df.index`, no?